src/components/learner-credit-management/data/utils.js

```diff
@@ -628,15 +628,20 @@ export const getNormalizedEnrollByDate = (enrollBy) => {
  *  - If hasEnrollBy, we return assignments with enroll before the soonest of the two date: The subsidy expiration
  *    date - refund threshold OR today offset by the 90-day allocation threshold for an assignment denoted as
  *    isEligibleForEnrollment
- *  - If isLateRedemptionEnabled, we consider only the isLateEnrollmentEligible field returned by Algolia for
+ *  - If isLateRedemptionAllowed, we consider only the isLateEnrollmentEligible field returned by Algolia for
  *    each run.
  *
  *  Based on the above criteria, if isLateRedemptionAllowed is false, filter on if the course run isActive AND
  *  isEligibleForEnrollment
  *
- *  We transform the assignedCourseRuns data to normalize the start and enrollby dates based on the functions
+ *  The main purpose of the filter is to ensure that course runs for a
+ *  course are within the enterprises LC subsidy duration
+ *  The inclusion of the increased sensitivity reduces the chance of a specific run
+ *  (which may be allocated but not accepted) falling outside the date range of the subsidy expiration date.
```

Comment:
IMO this doesn't directly speak to the reason for making the run visibility sensitive to the assignment expiration date. Maybe mention the specifics about how it's a product requirement or contractual requirement?

Reply:
The previously implemented 90-day assignment expiration date is now removed, after clarifying with product/Joe.